pkgs/applications/networking/browsers/vivaldi/ffmpeg-codecs.nix

```diff
@@ -1,30 +1,32 @@
-{ dpkg, fetchurl, lib, stdenv }:
+{ squashfsTools, fetchurl, lib, stdenv }:
 
+# This derivation roughly follows the update-ffmpeg script that ships with the official Vivaldi
+# downloads at https://vivaldi.com/download/
 stdenv.mkDerivation rec {
   pname = "chromium-codecs-ffmpeg-extra";
-  version = "112.0.5615.49";
+  version = "111306";
 
   src = fetchurl {
-    url = "https://launchpadlibrarian.net/660647727/${pname}_${version}-0ubuntu0.18.04.1_amd64.deb";
-    sha256 = "sha256-mHjvjRG+toRcsOMca+JPXNZPgyQROH2qtSpBPHLmt3s=";
+    url = "https://api.snapcraft.io/api/v1/snaps/download/XXzVIXswXKHqlUATPqGCj2w2l7BxosS8_34.snap";
+    sha256 = "sha256-Dna9yFgP7JeQLAeZWvSZ+eSMX2yQbX2/+mX0QC22lYY=";
   };
 
-  buildInputs = [ dpkg ];
+  buildInputs = [ squashfsTools ];
 
   unpackPhase = ''
-    dpkg-deb -x $src .
+    unsquashfs -dest . $src
   '';
 
   installPhase = ''
-    install -vD usr/lib/chromium-browser/libffmpeg.so $out/lib/libffmpeg.so
+    install -vD chromium-ffmpeg-${version}/chromium-ffmpeg/libffmpeg.so $out/lib/libffmpeg.so
   '';
 
   meta = with lib; {
     description = "Additional support for proprietary codecs for Vivaldi";
     homepage    = "https://ffmpeg.org/";
     sourceProvenance = with sourceTypes; [ binaryNativeCode ];
     license     = licenses.lgpl21;
-    maintainers = with maintainers; [ betaboon cawilliamson lluchs ];
+    maintainers = with maintainers; [ betaboon cawilliamson lluchs fptje ];
```

Comment:
Can we remove some inactive maintainers?

Reply:
I've removed @lluchs, as their last commit is from 2018. Both @betaboon and @cawilliamson have made commits in the past year.

I've only removed lluchs from this file.  They are also the sole maintainer of cups drivers and the first (out of two) of tectonic. Should I remove them from the entirety of nixpkgs in this PR or just from this file?

Reply:
lluchs at least did a PR in May, I just wanted to clean this package a bit up because the other 3 maintainer are not that eager with updates.